README.md

```diff
@@ -36,8 +36,8 @@ Device groups represent collection of Hubitat devices that can be accessed by us
 User groups represent collection of Telegram users that have access to device groups. User groups can contain any number of Telegram user ids (those with no user ids are ignored) and reference any number of device groups. User groups with an `access_level` set to:
 * `NONE`: cannot use any commands. Useful to disable a user group.
 * `DEVICE`: can use device commands e.g., `/list`, `/on`, `/off`, `/open`, `/close`, `/dim`, `/status`, `/commands`.
-* `HSM`: can use device commands and also [Hubitat Safety Monitor](https://docs.hubitat.com/index.php?title=Hubitat%C2%AE_Safety_Monitor_Interface) commands e.g, `/mode`, `/arm`, `/events`.
-* `ADMIN`: can use device, hsm and also admin commands e.g., `/users`, `/groups`, `/info`, `refresh`. In addition some commands have more detailed output (e.g., `/list`, `/status`).
+* `SECURITY`: can use device commands, the `/arm` command for [Hubitat Safety Monitor](https://docs.hubitat.com/index.php?title=Hubitat%C2%AE_Safety_Monitor_Interface), the `/mode` command to view and change the mode, the `/events` command to see a device's history, and act on locks with `/lock`, `/unlock`.
+* `ADMIN`: can use the device, security and also admin commands e.g., `/users`, `/groups`, `/info`, `refresh`. In addition some commands have more detailed output (e.g., `/list`, `/status`).
```

Comment:
I cannot make the sense out of `can use the device, security and`

Reply:
should make more sense now